server/src/infra/ldap_handler.rs

```diff
@@ -1523,6 +1565,144 @@ mod tests {
             ]
         );
     }
+    #[tokio::test]
+    async fn test_search_wildcards() {
+        use chrono::TimeZone;
+        let mut mock = MockTestBackendHandler::new();
+
+        mock.expect_list_users().times(..).returning(|_| {
```

Comment:
Can't you just not write `.times`?

Reply:
I'm not sure about this, I'll try it.
Might also be possible to omit times.